src/site/markdown/server/manage-webadmin.md

```diff
@@ -1754,6 +1755,108 @@ Response codes:
  - 404: The mail queue does not exist
  - 500: Internal error
 
+## Administrating DLP Configuration
+Webadmin allows Data Lost Prevention (DLP) configuration
```

Comment:
I'm quite sad, this desrciption is very short. I think we can at the very least re-use the explanations from swagger docs:

```
DLP (stands for Data Lost Prevention) is supported by James. A DLP matcher will, on incoming emails, 
execute regular expressions on email sender, recipients or content, in order to report suspicious emails to
an administrator. WebAdmin can be used to manage these DLP rules on a per sender-domain basis.
```